api/backend/duckdb.py

```diff
@@ -22,55 +22,56 @@
             l_comment VARCHAR NOT NULL);"""
 
 
-def ConnectDuckDB():
-    con = duckdb.connect()
-    con.install_extension("substrait")
-    con.load_extension("substrait")
-    con.install_extension("tpch")
-    con.load_extension("tpch")
-    con.execute(query=schema_lineitem)
-    logger.success("DuckDb initialized successfully")
-    return con
+class DuckDBConnection:
+    def __init__(self):
+        self.connection = None
 
+    def connect(self):
+        self.connection = duckdb.connect()
+        self.connection.install_extension("substrait")
+        self.connection.load_extension("substrait")
+        self.connection.install_extension("tpch")
+        self.connection.load_extension("tpch")
+        self.connection.execute(query=schema_lineitem)
```

Comment:
Creating the table doesn't seem like it belongs in the connect method?  maybe have a separate one just for the table creation?